src/errors/test-run/templates.js

```diff
@@ -386,4 +386,8 @@ export default {
     [TEST_RUN_ERRORS.cannotRestoreChildWindowError]: () => `
         Failed to restore connection to window within the allocated timeout.
     `,
+
+    [TEST_RUN_ERRORS.executionTimeoutExceeded]: err => {
+        return `${err.scope} timeout of ${err.timeout}ms exceeded.`;
```

Comment:
This message is not clear for a customer.
you need to say what kind of timeout is exceed.
Something like that:
`Test execution timeout is exceed (2000 ms).`

Also check the all texts with the technical writer.

Reply:
Scope means here: Test, Run.

Agreed on text with the technical writer.